packages/react-form/src/hooks/test/form.test.tsx

```diff
@@ -156,18 +160,18 @@ describe('useForm', () => {
         .find(TextField, {label: 'title'})!
         .trigger('onChange', 'tortoritos, the chip for turtles!');
 
-      wrapper
-        .find('button', {type: 'submit'})!
-        .trigger('onClick', clickEvent());
+      wrapper.act(() => {
+        wrapper
```

Comment:
I don't think you should need to wrap this in an act?

Reply:
Removing fails the test actually. It can't find the element.